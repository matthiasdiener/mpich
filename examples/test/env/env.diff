Differences in sigchk
2,10d1
< Signal SIGHUP has been changed
< Signal SIGINT has been changed
< Signal SIGQUIT has been changed
< Signal SIGFPE has been changed
< Signal SIGSEGV has been changed
< Signal SIGPIPE has been changed
< Signal SIGALRM has been changed
< Signal SIGTERM has been changed
< Signal SIGIO has been changed
