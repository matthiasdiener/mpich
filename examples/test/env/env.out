Timer tests
16 at [552d0], id = 0 p4_utils.c[486]
46084 at [47d18], id = 0 procgroup.c[12]
4096 at [46cc8], id = 0 p4_alloc.c[25]
8 at [46c70], id = 0 p4_MD.c[232]
20 at [46c10], id = 0 p4_alloc.c[20]
84 at [46b70], id = 0 p4_alloc.c[8]
8 at [46b18], id = 0 p4_MD.c[232]
8 at [46ac0], id = 0 p4_MD.c[232]
27892 at [3fd80], id = 0 p4_MD.c[232]
The maximum space allocated was 81716 bytes [0]
Basic attributes test
16 at [552c0], id = 0 p4_utils.c[486]
46084 at [47d08], id = 0 procgroup.c[12]
4096 at [46cb8], id = 0 p4_alloc.c[25]
8 at [46c60], id = 0 p4_MD.c[232]
20 at [46c00], id = 0 p4_alloc.c[20]
84 at [46b60], id = 0 p4_alloc.c[8]
8 at [46b08], id = 0 p4_MD.c[232]
8 at [46ab0], id = 0 p4_MD.c[232]
27892 at [3fd70], id = 0 p4_MD.c[232]
The maximum space allocated was 81716 bytes [0]
Error handling test
*** Error Handling ***
    Three error messages (from two errors) are expected
which should both show an error class of 13
(first) 13 : Invalid group passed to function
(errhandler) 13 : Invalid group passed to function
(second) 13 : Invalid group passed to function
24 at [57488], id = 0 util_hbt.c[94]
24 at [57420], id = 0 util_hbt.c[94]
24 at [573b8], id = 0 util_hbt.c[94]
28 at [57350], id = 0 attr_util.c[193]
28 at [572e8], id = 0 attr_util.c[193]
28 at [57280], id = 0 attr_util.c[193]
24 at [57218], id = 0 util_hbt.c[94]
24 at [571b0], id = 0 util_hbt.c[94]
24 at [57148], id = 0 util_hbt.c[94]
28 at [570e0], id = 0 attr_util.c[193]
28 at [57078], id = 0 attr_util.c[193]
28 at [57010], id = 0 attr_util.c[193]
16 at [56f10], id = 0 util_hbt.c[76]
16 at [554d0], id = 0 p4_utils.c[486]
46084 at [47f18], id = 0 procgroup.c[12]
4096 at [46ec8], id = 0 p4_alloc.c[25]
8 at [46e70], id = 0 p4_MD.c[232]
20 at [46e10], id = 0 p4_alloc.c[20]
84 at [46d70], id = 0 p4_alloc.c[8]
8 at [46d18], id = 0 p4_MD.c[232]
8 at [46cc0], id = 0 p4_MD.c[232]
27892 at [3ff80], id = 0 p4_MD.c[232]
The maximum space allocated was 81916 bytes [0]
Signal test
Signal SIGINT has been changed
Signal SIGFPE has been changed
Signal SIGBUS has been changed
Signal SIGSEGV has been changed
16 at [55320], id = 0 p4_utils.c[486]
46084 at [47d68], id = 0 procgroup.c[12]
4096 at [46d18], id = 0 p4_alloc.c[25]
8 at [46cc0], id = 0 p4_MD.c[232]
20 at [46c60], id = 0 p4_alloc.c[20]
84 at [46bc0], id = 0 p4_alloc.c[8]
8 at [46b68], id = 0 p4_MD.c[232]
8 at [46b10], id = 0 p4_MD.c[232]
27892 at [3fdd0], id = 0 p4_MD.c[232]
The maximum space allocated was 81716 bytes [0]
